.github/terraform/terraform.tfvars

```diff
@@ -0,0 +1 @@
+prefix = SET_PREFIX
```

Comment:
Could you add other variables with their default values? It might help understand the setup and maintain/update it easily.

Reply:
I will add them.